OpenRA.Mods.Common/Widgets/Logic/Ingame/ObserverStatsLogic.cs

```diff
@@ -220,6 +220,7 @@ ScrollItemWidget ProductionStats(Player player)
 
 			template.Get<ObserverProductionIconsWidget>("PRODUCTION_ICONS").GetPlayer = () => player;
 			template.Get<ObserverSupportPowerIconsWidget>("SUPPORT_POWER_ICONS").GetPlayer = () => player;
+			template.IgnoreChildMouseOver = false;
```

Comment:
This can't be currently set in yaml, because it is overriden in ScrollItemWidget(ScrollItemWidget other).